src/components/MultiSelect/MultiSelect.js

```diff
@@ -194,18 +194,17 @@ export default class MultiSelect extends React.Component {
                     }).map((item, index) => {
                       const itemProps = getItemProps({ item });
                       const itemText = itemToString(item);
+                      var itemIndex = selectedItem
+                        .map(function(selected) {
+                          return selected.id;
+                        })
+                        .indexOf(item.id);
                       const isChecked =
-                        selectedItem.findIndex(
-                          selected => item.id === selected.id
-                        ) !== -1;
+                        typeof selectedItem[itemIndex] !== 'undefined' || false;
```

Comment:
Thank you for your quick change @zerostatic! Given you mimicked `findIndex`, do you want to keep `!== -1` comparison?